Tribler/community/bartercast4/statistics.py

```diff
@@ -144,7 +144,7 @@ def close(self):
  UNIQUE (type, peer));
 
 CREATE TABLE option(key TEXT PRIMARY KEY, value BLOB);
-INSERT INTO option(key, value) VALUES('database_version', '""" + str(LATEST_VERSION) + """');
+INSERT INTO option(key, value) VALUES('database_version', '""" + str(LATEST_VERSION) + u"""');
```

Comment:
Also, if you find bugs, send them to `next` instead (even if it's a single line PR) otherwise they will not get into the next release.